css/main.css

```diff
@@ -1,3 +1,190 @@
+}
+/* hack how to make humburger logo using checkbox */
+input[type="checkbox"] {
+  display: none;
+}
+
+.looking {
+  padding: 15px;
+}
+/* change color of basket */
+.basket {
+  /* background-color:var(--red-brown); */
+  padding-top: 6px;
+  margin-right: 8px;
+}
+
+.search {
+  /* position: static; */
+  display: flex;
+  height: 3rem;
+  width: 100%;
+  min-height: auto;
+  justify-content: space-between;
+}
+
+.search__input {
+  border: none;
+  width: 12rem;
+  margin-left: 8px;
+  padding-left: 20px;
+}
+
+.info_main {
+  background-color: var(--light-grey);
+  width: 100%;
+  height: 100px;
+  display: grid;
+  grid-template-columns: repeat(4, auto);
+  
+}
+
+.info {
+  grid-column: 1 / span 3;
+}
+
+.info a {
+ font-size: 30px;
+ font-family:var(--subheading); 
+ color: var(--red-brown);
+ text-decoration: none;
+ 
+}
+
+.info p{
+  font-size: 20px;
+ font-family:var(--subheading); 
+ color: var(--red-brown);
+ margin-top: 15px;
+ letter-spacing: 3px;
+ /* font-weight: 900; */
+}
+.humburger {
+  width:30px;
+  height:30px;
+  color:var(--red-brown);
+}
+
+.menu {
+  grid-column: 4/4;
+  display:none;
+}
+.dropdown {
+    display: block;
+    position: absolute; 
+}
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  background: var(--light-grey);
+}
+
+ul li {
+  display: block;
+  position: relative;
+  line-height: 21px;
+  text-align: left;
+}
+
+ul li a {
+  display: block;
+  padding: 10px;
+  text-decoration: none;
+  color: var(--red-brown);
+}
+
+.dropdown:hover {
+  display: block;
+  cursor:pointer;
+  
+}
+
+.main {
+  background-image: url("../img/background\ for\ last\ week\ html.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-attachment: fixed;
+  background-size: cover;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  width: 100%;
+  height: 620px;
+}
+
+.estb_2023 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-evenly;
+  color: var(--white);
+  font-size: 20px;
+  letter-spacing: 4px;
+}
+
+.centercontent {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  padding-top: 100px;
+}
+.brand_logo {
+  color: var(--white);
+  margin: 0 20px;
+}
+
+.brand_name h1 {
+  font-family: "Fjalla One", sans-serif;
+  color: var(--white);
+  font-size: 44px;
+  letter-spacing: 0.17em;
+  margin-bottom: 20px;
+  padding-left:30px;
+}
+
+.brand_name p {
+  font-family: "Outfit", sans-serif;
+  color: var(--white);
+  font-size: 16px;
+  letter-spacing: 0.35em;
+  /* margin-bottom: 43px;  use margin-top for button*/
+  margin: 0 65px;
+  text-align: center;
+}
+
+.shop {
+  background-color: var(--red-brown);
+  color: var(--white);
+  padding: 15px 30px;
+  margin-top: 43px;
+  font-size: 18px;
+  font-family: "Outfit", sans-serif;
+  letter-spacing: 2px;
+}
+
+.wrapper_photo {
+  display: grid;
+  
+  gap:20px;
+}
+
+.photo img {
+  width: 100%;
+  height:auto;
```

Comment:
`auto` is the default value of `height`, so you may decide not to write it here.